nix/default.nix

```
@@ -18,8 +18,8 @@
   glibc,
   lld,
 }:
-assert lib.assertMsg (lib.versionAtLeast "1.88.0" pkgs.rustc.version)
-  "Wild requires at least Rust 1.88.0, this instance of nixpkgs has Rust ${pkgs.rustc.version}";
+assert lib.assertMsg (lib.versionAtLeast "1.89.0" pkgs.rustc.version)
```

```suggestion
assert lib.assertMsg (lib.versionAtLeast pkgs.rustc.version "1.89.0")
```
This call was incorrect, even before this PR. `versionAtLeast` checks that the first parameter is at least the second, so it would require that the rustc version was *less than or equal to* 1.88.0 (prior to this PR), instead of the reverse.

Relevant docs: https://ryantm.github.io/nixpkgs/functions/library/strings/#function-library-lib.strings.versionAtLeast